modules/assisted-installer-using-the-assisted-installer.adoc

```diff
@@ -0,0 +1,42 @@
+// This is included in the following assemblies:
+//
+// installing-on-prem-assisted.adoc
+:_content-type: CONCEPT
+
+[id="using-the-assisted-installer_{context}"]
+= Using the Assisted Installer
+
+The {product-title} link:https://console.redhat.com/openshift/assisted-installer/clusters/~new[{ai-full}] is a user-friendly installation solution offered on the link:http://console.redhat.com[Red Hat Hybrid Cloud Console]. The {ai-full} supports the various deployment platforms with a focus on bare metal and vSphere.
+
+The {ai-full} provides installation functionality as a service. This software-as-a-service (SaaS) approach has the following advantages:
+
+* *Web user interface:* The web user interface performs cluster installation without the user having to create the installation configuration files manually.
+* *No bootstrap node:* A bootstrap node is not required when installing with the {ai-full}. The bootstrapping process executes on a node within the cluster.
+* *Hosting:* The {ai-full} hosts:
+  - Ignition files
+  - The installation configuration
+  - A discovery ISO
+  - The installer
+* *Streamlined installation workflow:* Deployment does not require in-depth knowledge of {product-title}. The {ai-full} provides reasonable defaults and provides the installer as a service, which:
+  - Eliminates the need to install and run the {product-title} installer locally
+  - Ensures the latest version of the installer up to the latest tested z-stream releases. Older versions remain available, if needed
+  - Enables building automation through the API without the need to run the {product-title} installer locally
+* *Advanced networking:* The {ai-full} supports IPv4/IPv6 dual stack networking, NMState-based static IP addressing, and an HTTP/S proxy.
+* *Pre-installation validation:* The {ai-full} validates the configuration before installation to ensure a high probability of success. Validation includes:
+  - Ensuring network connectivity
+  - Ensuring sufficient network bandwidth
+  - Ensuring connectivity to the registry
+  - Ensuring time synchronization between cluster nodes
+  - Verifying that the cluster nodes meet the minimum hardware requirements
+  - Validating the installation configuration parameters
+* *REST API:* The {ai-full} has a REST API, enabling automation.
+
+The {ai-full} supports installing {product-title} on premises in a connected environment, including with an optional HTTP/S proxy.  It can install:
+
+* Highly available {product-title} or Single Node OpenShift (SNO)
+* On bare metal or vSphere with full platform integration, or other virtualization platforms without integration
```

Comment:
This was a little bit hard to parse, because it describes a place to install something vs a thing to be installed like the other items. For clarity, it might make sense to have it begin without the preposition, so that it matches more of what the other items are trying to say:
```suggestion
* {product-title} on bare metal or vSphere with full platform integration, or other virtualization platforms without integration
```